ru/turing/courses/lesson5/Main.java

```diff
@@ -1,7 +1,185 @@
 package ru.turing.courses.lesson5;
 
+import java.util.*;
+
 public class Main {
     public static void main(String[] args) {
         //todo вставить сюда сортировки пузырьком, сделать для каждой свой метод
+        int[] mass = new int[]{8, 4, 9, 15, 26, -1, 0};
```

Comment:
```suggestion
        int[] intArray = new int[]{8, 4, 9, 15, 26, -1, 0};
```
Нейминг должен быть со смыслом